qt/app.py

```diff
@@ -706,13 +706,29 @@ def _create_main_toolbar(self):
             self.act_shutdown,
         ]
 
-        toolbar.addActions(actions_for_toolbar)
+        # Add each action to toolbar
+        for act in actions_for_toolbar:
+            toolbar.addAction(act)
+
+            # Assume an explicit tooltip if it is different from "text()".
+            # Note that Qt use "text()" as "toolTip()" by default.
+            if act.toolTip() != act.text():
+
+                if toolbar.layoutDirection() == 1:
+                    # RTL/BIDI languange like Hebrew
+                    button_tip = f'{act.toolTip()} :{act.text()}'
+                else:
+                    # LTR language (e.g. English)
+                    button_tip = f'{act.text()}: {act.toolTip()}'
+
+                toolbar.widgetForAction(act).setToolTip(button_tip)
```

Comment:
I would have been much easier to "inject" the tooltip if Qt5 would have added the tooltip text to the `QAction` constructor. Now you have to loop over each action "just" to set the tooltip ;-)